distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/cli/RemovePluginActionTests.java

```diff
@@ -141,14 +141,18 @@ public void testRemoveMultiple() throws Exception {
         assertRemoveCleaned(env);
     }
 
+    private static Version minimumCompatibleVersion(Version v) {
+        return Version.fromString((v.major - 1) + ".0.0");
+    }
+
     public void testRemoveOldVersion() throws Exception {
         Version previous = VersionUtils.getPreviousVersion();
-        if (previous.before(Version.CURRENT.minimumIndexCompatibilityVersion())) {
```

Comment:
I don't think plugins actually need to reference index compatibility here?

Reply:
The test is trying to mimic a plugin that was built against the oldest version that supports a full cluster restart. This if statement is for an edge case when we have just branched a new major version. In that case, calling VersionUtils.getPreviousVersion will return an unreleased version (the unreleased N.last).

In any case, I think this test is rather silly. Nothing about removing a plugin cares about the version of Elasticsearch the plugin was built for. I suggest to just remove the test entirely.

Reply:
Removed